src/com/clover/remote/client/transport/websocket/WebSocketCloverTransport.ts

```diff
@@ -163,7 +163,11 @@ export abstract class WebSocketCloverTransport extends CloverTransport implement
 	 * @returns {number} negative 1 (-1)
      */
 	public sendMessage(message: string): number {
-		this.messageQueue.push(message);
+		if (!this.shutdown) {
+			this.messageQueue.push(message);
+		} else {
+			this.logger.debug('In process of shutting down, ignoring ' + message);
```

Comment:
Just an FYI - you can use template literals here `In process of shutting down, ignoring ${message}`.